governance.md

```diff
@@ -153,49 +289,135 @@ are enumerated below.
 
 ## Other duty-specific github roles:
 - [**Github Org Owner**](https://github.com/orgs/kubernetes/people?utf8=%E2%9C%93&query=%20role%3Aowner):
-  can create repos, do ~any github action; the number of
+  can create repos, and do any github action; the number of
```

Comment:
People who've been on the project a long time -- Google and Redhat today, yes. Sometimes Mirantis and I think CoreOS. It's on an as-needed basis.